README.md

```diff
@@ -169,8 +169,49 @@ Using pgadmin4 we can inspect the postgres database's schemas and tables (helpfu
 8. You'll probably want to look at Schemas > (pick a schema) > Tables
 
 
-## Contributing
+### Enabling Google Sign In (Optional)
 
-See [CONTRIBUTING.md](https://github.com/bytedeck/bytedeck/blob/develop/CONTRIBUTING.md) if you plan to contribute code!
 
+Here are the steps, assuming that you now have a functional tenant:
+
+1. Obtain Google credentials: https://developers.google.com/workspace/guides/create-credentials#oauth-client-id
+2. Make sure that in the Authorized URIs, add `http://hackerspace.localhost.net:8000/accounts/google/login/callback/`. We will explain why we are using `localhost.net` later but for now, just add this.
+3. Go to Social Applications: http://hackerspace.localhost:8000/admin/socialaccount/socialapp/
+4. Click Add Social Application
+5. Fill in `Client Id` and `Secret Key`. And then add the `Available Sites` to `Chosen Sites`
+6. Click Save
+7. Go to your Site Configuration: http://hackerspace.localhost:8000/config/ and click `Enable sign-in via Google`
+8. Done
+
+When you are developing locally, Google won't allow you to add `http://hackerspace.localhost:8000/accounts/google/login/callback/` in the Authorized URIs. So we need a way to bypass this in our local machine by mapping
+our localhost to `localhost.net` so we can access our tenant via `http://hackerspace.localhost.net:8000`.
+
+We need to modify our hosts file aka `/etc/hosts`.
```

Comment:
Where is this?  in docker config somewhere?

Reply:
I'll update the docs to clarify. This is only done when developing locally and should be done in the host machine